src/games/strategy/engine/data/Route.java

```diff
@@ -96,24 +96,6 @@ public static Route join(final Route r1, final Route r2) {
     return joined;
   }
 
```

Comment:
Clever algorithms are fun, but I would ask:
- is this really that difficult to recreate if needed?
- Given there are no usages and thus no requirements around this function, how do we know that some version of this needed in the future won't be just radically different. For example, this takes a very naive approach and just finds *a* route between two points. That route might have been taken going the wrong way around the world through impassible territories.

With that said, I don't really care so much and can restore the bit of code and mark the method as private.